### PR TITLE
edit pass: Protobuf chapters

### PR DESCRIPTION
**Pull request by ShawnJackson**
@shirhatti, @mairaw - Please review the copy edit to these articles and let me know if you have any feedback. This is for Azure DevOps work request [1646344](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1646344).

Contributes to #15889.

### CODE REVIEW

File: docs/architecture/grpc-for-wcf-developers/protobuf-any-oneof.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-title: Protobuf Any and Oneof fields for variant types - gRPC for WCF Developers
+title: Protobuf Any and Oneof fields for variant types - gRPC for WCF developers
 description: Learn how to use the Any type and the Oneof keyword to represent variant object types in messages.
 ms.date: 09/09/2019
 ---
 
 # Protobuf Any and Oneof fields for variant types
 
-Handling dynamic property types (that is, properties of type `object`) in WCF is complicated. Serializers must be specified, [KnownType](xref:System.Runtime.Serialization.KnownTypeAttribute) attributes must be provided, and so on.
+Handling dynamic property types (that is, properties of type `object`) in Windows Communication Foundation (WCF) is complicated. For example, you must specify serializers, and you must provide [KnownType](xref:System.Runtime.Serialization.KnownTypeAttribute) attributes.
```
Comment (Youssef1313) [inline docs/architecture/grpc-for-wcf-developers/protobuf-any-oneof.md]:
```suggestion
Handling dynamic property types (that is, properties of type `object`) in Windows Communication Foundation (WCF) is complicated. For example, you must specify serializers, and provide [KnownType](xref:System.Runtime.Serialization.KnownTypeAttribute) attributes.
```

**Review by mairaw — COMMENTED** [review]
The edits look good to me @ShawnJackson. Left some nit comments.

@shirhatti please review and sign off.

File: docs/architecture/grpc-for-wcf-developers/protobuf-enums.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-title: Protobuf enumerations - gRPC for WCF Developers
+title: Protobuf enumerations - gRPC for WCF developers
 description: Learn how to declare and use enumerations in Protobuf.
 ms.date: 09/09/2019
 ---
 
 # Protobuf enumerations
 
-Protobuf supports enumeration types, as seen in the previous section where an enum was used to determine the type of a `oneof` field. You can define your own enumeration types and Protobuf will compile them to C# enum types. Since Protobuf can be used with different languages, the naming conventions for enumerations are different from the C# conventions. However, the code generator is clever and converts the names to the traditional C# case. If the Pascal-case equivalent of the field name starts with the enumeration name, then it's removed.
+Protobuf supports enumeration types. You saw this support in the previous section, where an enum was used to determine the type of a `oneof` field. You can define your own enumeration types, and Protobuf will compile them to C# enum types.
```
[the remark this answers is not in the document]
Comment (ShawnJackson) [reply docs/architecture/grpc-for-wcf-developers/protobuf-enums.md]: Fixed!

**Comment by tfosmark** [comment]
@shirhatti Hi, this edit pass is still waiting on your signoff. Can you help wrap this up? Thanks!

**Review by shirhatti — APPROVED** [review]

File: docs/architecture/grpc-for-wcf-developers/protobuf-data-types.md [hunk 3]
```diff
@@ -177,7 +185,7 @@ namespace CustomTypes
 ```
 
 > [!IMPORTANT]
-> Whenever you use custom utility message types like this, you **must** document them with comments in the `.proto` so that other developers can implement conversion to and from the equivalent type in their own language or framework.
+> Whenever you use custom tool message types like this, you *must* document them with comments in `.proto`. Other developers can then implement conversion to and from the equivalent type in their own language or framework.
```
[the remark this answers is not in the document]
Comment (ShawnJackson) [reply docs/architecture/grpc-for-wcf-developers/protobuf-data-types.md]: I've committed the change.

**Pull request merged**